exercises/two-fer/lib/example.dart

```diff
@@ -0,0 +1,5 @@
+class TwoFer {
+  String twofer(String name) {
+    return "One for ${name is String ? name : 'you'}, one for me.";
```

Comment:
This exercise could really make good use of default parameters. You might have to add a test where no parameters are passed into the function.